lib/msf/core/feature_manager.rb

```diff
@@ -103,6 +104,13 @@ class FeatureManager
         default_value: false,
         developer_notes: 'To be enabled by default after appropriate testing'
       }.freeze,
+      {
+        name: SHOW_SUCCESSFUL_LOGINS,
+        description: 'When enabled scanners/login modules will return a table off successful logins once the module completes',
+        requires_restart: false,
+        default_value: true,
```

Comment:
```suggestion
        default_value: false,
```

We shouldn't enable this by default until we've dogfooded it for a bit